src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -997,7 +999,7 @@ class HookUtils(object):
                     if key.startswith('mic'):
                         mics.append(key[3:])
                     elif key.startswith('nvidia'):
-                        gpus.append(key[6:])
+                        gpus.append(node.devices['gpu'][key]['uuid'])
```

Comment:
I had some concern over whether the prior use of key[6:] was done for a reason. Looking at the rest of the changes, I'm convinced that you are doing the right thing here.